ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -94,18 +97,26 @@ http {
         }
 
         location /blackbox.tar.gz {
+            limit_req zone=mylimit burst=20 nodelay;
+            
             return 301 https://github.com/apache/incubator-openwhisk-runtime-docker/releases/download/sdk%400.1.0/blackbox-0.1.0.tar.gz;
         }
         # leaving this for a while for clients out there to update to the new endpoint
         location /blackbox-0.1.0.tar.gz {
+            limit_req zone=mylimit burst=20 nodelay;
+            
             return 301 /blackbox.tar.gz;
         }
 
         location /OpenWhiskIOSStarterApp.zip {
+            limit_req zone=mylimit burst=20 nodelay;
```

Comment:
As we only return a 301 to a github address: Do we really need to throttle this request (and the others with 301)?